File: src/main/java/net/hectus/neobb/turn/person_game/categorization/WinConCategory.java
package net.hectus.neobb.turn.person_game.categorization;

import net.kyori.adventure.text.format.NamedTextColor;
import net.kyori.adventure.text.format.TextColor;
import org.bukkit.Material;

public interface WinConCategory extends AttackCategory {
    @Override
    default TextColor categoryColor() {
        return NamedTextColor.DARK_RED;
    }

    @Override
    default Material categoryItem() {
        return Material.PURPLE_WOOL;
    }

    @Override
    default String categoryName() {
        return "win_con";
    }
}

File: src/main/kotlin/net/hectus/neobb/turn/person_game/categorization/WinConCategory.kt
package net.hectus.neobb.turn.person_game.categorization

import net.kyori.adventure.text.format.NamedTextColor
import net.kyori.adventure.text.format.TextColor
import org.bukkit.Material

interface WinConCategory : AttackCategory {
    override val categoryColor: TextColor get() = NamedTextColor.DARK_RED

    override val categoryItem: Material get() = Material.PURPLE_WOOL
    override val categoryName: String get() = "win_con"
}
